create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils/check-requirements.py

```diff
@@ -0,0 +1,61 @@
+import subprocess
+import os
+import click
+from functools import partial
+from pathlib import Path
+
```

Comment:
Here u can read reasons why need to write main function for python's script.

https://stackoverflow.com/questions/45129728/what-is-the-best-practice-for-running-a-python-script?answertab=votes#tab-top

I can add from me:

 - reusing a code 
 - a better structure of code

Of course, this does not affect the final result, but I highly recommend using this propose with `main ()` function 😀